packages/stark-core/src/modules/session/services/session.service.ts

```diff
@@ -33,11 +33,12 @@ import {
 } from "../actions";
 import { StarkHttpStatusCodes } from "../../http/enumerators";
 import { StarkHttpHeaders } from "../../http/constants";
-import { starkSessionExpiredStateName } from "../../../common/routes";
 import { StarkCoreApplicationState } from "../../../common/store";
 import { selectStarkSession } from "../reducers";
 import { StarkConfigurationUtil } from "../../../util/configuration.util";
 import { StarkValidationErrorsUtil } from "../../../util";
+// FIXME
+import { starkSessionExpiredStateName } from "../routes";
```

Comment:
I think we should decouple this service from the "session routes" and create a new property `sessionExpiredStateName` in the Appconfig. In that way, the "session routes" can be safely placed in the Session UI module.

The same needs to be applied to the AppLogout component which would need the `starkSessionLogoutState`

What do you think?